### PR TITLE
SFMC Integration

### PR DESCRIPTION
**Pull request by ruchiramoitra**
## Description of the change

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: v0/destinations/sfmc/data/SFMCInsertIdentifyContactsConfig.json [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    "required": false
+  },
+  {
+    "destKey": "User Agent",
+    "sourceKeys": "context.userAgent",
+    "required": false
+  },
+  {
+    "destKey": "IP Address",
+    "sourceKeys": "context.ip",
```
Comment (arnab-p) [inline v0/destinations/sfmc/data/SFMCInsertIdentifyContactsConfig.json:54]:
```suggestion
    "sourceKeys": ["context.ip", "request_ip"],
```

File: v0/destinations/sfmc/data/SFMCInsertIdentifyContactsConfig.json [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+  {
+    "destKey": "Device Type",
+    "sourceKeys": "context.device.type",
+    "required": false
+  },
+  {
+    "destKey": "Ad Tracking Enabled",
+    "sourceKeys": "context.device.adTrackingEnabled",
+    "required": false
+  },
```
Comment (arnab-p) [inline v0/destinations/sfmc/data/SFMCInsertIdentifyContactsConfig.json:86]: Duplicate entry

File: v0/destinations/sfmc/data/SFMCInsertIdentifyContactsConfig.json [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+  {
+    "destKey": "Screen Width",
+    "sourceKeys": "context.screen.width",
+    "required": false
+  },
+  {
+    "destKey": "",
+    "sourceKeys": [
+      "context.traits",
+      "traits"
```
Comment (arnab-p) [inline v0/destinations/sfmc/data/SFMCInsertIdentifyContactsConfig.json:126]: Top level `traits` should be prioritized

File: v0/destinations/sfmc/data/SFMCInsertTrackContactsConfig.json [hunk 4]
```diff
@@ -0,0 +1,127 @@
+[
```
Comment (arnab-p) [inline v0/destinations/sfmc/data/SFMCInsertTrackContactsConfig.json:1]: Format file with 2 spaces

File: v0/destinations/sfmc/data/SFMCInsertTrackContactsConfig.json [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+      "required": false
+    },
+    {
+      "destKey": "User Agent",
+      "sourceKeys": "context.userAgent",
+      "required": false
+    },
+    {
+      "destKey": "IP Address",
+      "sourceKeys": "context.ip",
```
Comment (arnab-p) [inline v0/destinations/sfmc/data/SFMCInsertTrackContactsConfig.json:54]:
```suggestion
		"sourceKeys": ["context.ip", "request_ip"],
```

File: v0/destinations/sfmc/data/SFMCInsertTrackContactsConfig.json [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+    {
+      "destKey": "Device Type",
+      "sourceKeys": "context.device.type",
+      "required": false
+    },
+    {
+      "destKey": "Ad Tracking Enabled",
+      "sourceKeys": "context.device.adTrackingEnabled",
+      "required": false
+    },
```
Comment (arnab-p) [inline v0/destinations/sfmc/data/SFMCInsertTrackContactsConfig.json:86]: Duplicate entry

**Review by arnab-p — CHANGES_REQUESTED** [review]

File: v0/destinations/sfmc/data/SFMCInsertIdentifyContactsConfig.json (hunk 1 above)
Comment (ruchiramoitra) [reply v0/destinations/sfmc/data/SFMCInsertIdentifyContactsConfig.json:54]: done

File: v0/destinations/sfmc/data/SFMCInsertIdentifyContactsConfig.json (hunk 2 above)
Comment (ruchiramoitra) [reply v0/destinations/sfmc/data/SFMCInsertIdentifyContactsConfig.json:86]: done

**Pull request merged**